frontends/web/src/routes/lightning/receive/receive.tsx

```diff
@@ -159,7 +159,7 @@ export function Receive() {
             <Grid col="1">
               <Column>
                 <h1 className={styles.title}>{t('lightning.receive.subtitle')}</h1>
-                <span>{t('lightning.receive.amountSats.label')} ({<FiatConversion amount={invoiceAmount} noBtcZeroes />})</span>
+                <span>{t('lightning.receive.amountSats.label')} ({<FiatConversion alwaysShowAmounts amount={invoiceAmount} noBtcZeroes />})</span>
                 <Input
                   type="number"
                   min="0"
```

Comment:
there is one more FiatConversion component in the 'invoice' step under the QR code. Please also add `alwaysShowAmounts ` prop there